AsyncDisplayKit/ASCellNode.h

```diff
@@ -167,6 +161,19 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 - (void)cellNodeVisibilityEvent:(ASCellNodeVisibilityEvent)event inScrollView:(nullable UIScrollView *)scrollView withCellFrame:(CGRect)cellFrame;
 
+#pragma mark - UITableViewCell specific passthrough properties
+
+/*
```

Comment:
Can you add another state otherwise it will not show up in the Xcode documentation browser.